specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -647,29 +527,16 @@
             }
         },
         "ErrorResponse": {
+            "description": "Describes the format of Error response.",
             "type": "object",
             "properties": {
-                "error": {
-                    "type": "string",
+                "code": {
```

Comment:
Are you sure this is how it looks on the wire? Swagger spec needs to correctly and completely describe the API as it is on the wire. Just want to make sure that this is the right thing. Had we merged the PR with the previous change and if you would have sent another PR with this change then that would result in a breaking change in the SDK.

Hence please be very careful about how you describe the APIs in the swagger spec.